.github/workflows/if-nodejs-pr-testing.yml

```diff
@@ -76,7 +76,7 @@ jobs:
         run: npm run generate:assets --if-present
 
       # Run the test:md script and capture output
-      - if: steps.packagejson.outputs.exists == 'true'
+      - if: ${{ steps.packagejson.outputs.exists == 'true' && matrix.os == 'ubuntu-latest' }}
```

Comment:
Why are we restricting the tests to ubuntu os only?

Reply:
Because we are using the commands like `sed` to parse the errors, windows bash doesn't have that support